airflow/www/views.py

```diff
@@ -449,6 +490,8 @@ def get_int_arg(value, default=0):
         arg_search_query = request.args.get('search')
         arg_tags_filter = request.args.getlist('tags')
         arg_status_filter = request.args.get('status')
+        arg_sorting_key = request.args.get('sortBy')
+        arg_orderby_key = request.args.get('orderBy')
```

Comment:
These two params are two confusing/similar, and clash a bit with standard SQL meanings, since the sortBy value is the term after the `ORDER BY`.

Could we instead have `orderBy` and `orderDir` (direction).

(I also would have personally done it as `orderBy=dag_id` for asc and `orderBy=-dag_id` for desc. It means you can then order by multiple columns with `orderBy=col_a,-col_b` or `orderBy=col_a&orderBy=-col_b`. But feel free to leave it as two params, it's fine as it is)

Reply:
K, sortBy and orderBy as argument names is too confusing.

I'd like us to either:

a) change to the single column using `-` (`order_by=col` for asc, `order_by=-col` for descending.); or
b) change orderBy to `orderDir` (have I got this the right way around?)

a has the advantage that you could then sort by multiple columns (`order_by=col1,-col2`) for instance but I'd be okay not supporting that for now.

Reply:
Went with (B) for time being since it requires less changes and I'm already worried about rebases messing up other changes.